kivy/logger.py

```diff
@@ -115,8 +116,8 @@ def purge_logs(self, directory):
         if randint(0, 20) != 0:
             return
 
-        # Use config ?
-        maxfiles = 100
+        from kivy.config import Config
```

Comment:
Replacing several 'import kivy.Config' statements with a single one at the top was not possible:
`I/python  ( 5561): Traceback (most recent call last):
I/python  ( 5561):   File "main.py", line 10, in <module>
I/python  ( 5561):     from kivy.support import install_twisted_reactor
I/python  ( 5561):   File "/data/data/de.ibrieger.ibr/files/app/lib/python2.7/site-packages/kivy/__init__.py", line 38, in <module>
I/python  ( 5561):     from kivy.logger import Logger, LOG_LEVELS
I/python  ( 5561):   File "/data/data/de.ibrieger.ibr/files/app/lib/python2.7/site-packages/kivy/logger.py", line 62, in <module>
I/python  ( 5561):     from kivy.config import Config
I/python  ( 5561):   File "/data/data/de.ibrieger.ibr/files/app/lib/python2.7/site-packages/kivy/config.py", line 320, in <module>
I/python  ( 5561):     from kivy import kivy_config_fn
I/python  ( 5561): ImportError: cannot import name kivy_config_fn
`